maker/src/cfd.rs

```diff
@@ -961,6 +886,39 @@ where
     }
 }
 
+/// Source of offer rates used for rolling over CFDs.
+#[derive(Clone)]
+pub struct RatesChannel(MessageChannel<GetOffers, Option<MakerOffers>>);
```

Comment:
Maybe we could opt to implement this directly on the `offers` protocol.
In my latest changes to the `order` protocol I also added the possibility to fetch the latest offer, see: https://github.com/itchysats/itchysats/pull/2292/commits/eaa7016d98d4c0a1e7731a0442283ebbda1cbe4c

We might want to harmonize this, but we can do that on top of this PR.

Reply:
See https://github.com/itchysats/itchysats/issues/2546.